docs/it/week06/06-1.md

```diff
@@ -95,216 +99,232 @@ Le reti neurali convoluzionali svolgono bene le attività di rilevamento ed anch
+
+In ogni regione con punteggio elevato vi è probabilmente sotto una faccia. Se vengono rilevati più volti molto vicini al primo, significa che solo uno dovrebbe essere considerato corretto e il resto sbagliato. Con la soppressione dei non-massimi, prendiamo il punteggio più alto tra le *bounding boxes* sovrapposte e rimuoviamo gli altri. Il risultato sarà una unica *bounding box* nella posizione ottimale.
 
 
-<!-- 
+<!--
 ### Negative mining
 -->
+
 ### Estrazione negativa
 
-<!-- 
+<!--
 In the last section, we discussed how the model will run into a large number of false positives at test time as there are many ways for objects to appear similar to a face. No training set will include all the non-face objects that look like faces. We can mitigate this problem through negative mining. In negative mining, we create a negative dataset of non-face patches that the model detects as faces. The data is collected by running the model on inputs that are known to contain no faces. Then we retrain the detector using the negative dataset. We can repeat this process to increase the robustness of our model against false positives.
 -->
-Nell'ultima sezione, abbiamo discusso di come il modello si imbatterà in un gran numero di falsi positivi al momento del test poiché ci sono molti modi per far apparire un oggetto simile a una faccia. Nessun set di addestramento includerà tutti gli oggetti che sembrano volti ma non lo sono. Possiamo mitigare questo problema attraverso il mining negativo. Nel mining negativo, creiamo un set di dati negativo di patch che non contengono volti che il modello rileva erroneamente come facce. I dati vengono raccolti eseguendo il modello su input noti per non contenere volti. Quindi addestriamo nuovamente il rivelatore utilizzando il set di dati negativi. Possiamo ripetere questo processo per aumentare la solidità del nostro modello contro i falsi positivi.
 
+Nell'ultima sezione, abbiamo discusso di come il modello si possa imbattere in un numero elevato di falsi positivi al momento del test poiché ci sono molti modi per far apparire un oggetto che non è una faccia (e di consequenza l'algoritmo potrebbe scambiare un oggetto che non è una faccia per una faccia). Nessun set di addestramento potrà mai includere tutti gli oggetti che sembrano volti ma non lo sono. Possiamo mitigare questo problema attraverso il mining negativo. Nel mining negativo, creiamo un set di dati negativi formato da pezzi (*patches*) di immagine che non contengono volti che il modello rileva erroneamente come facce. I dati vengono raccolti eseguendo il modello su input noti per non contenere volti. Quindi addestriamo nuovamente il rivelatore utilizzando il set di dati negativi. Possiamo ripetere questo processo per aumentare la solidità del nostro modello contro i falsi positivi.
```

Comment:
Ho cambiato questa frase perché a me sembrava che i falsi positivi fossero dovuti al fatto che non ci sono abbastanza esempi di immagini senza facce. È così?

Reply:
No, no, era giusto prima.
I falsi positivi sono dovuti al fatto che la rete andrà semplicemente a cercare il colore della pelle per individuare una faccia. Di conseguenza, una mano è rilevata come una faccia. Quindi bisogna inserire mani (o qualsiasi altra non-faccia che il modello reputa essere una faccia) nel data set originario.
Se pensi questa spiegazione sia migliore, possiamo aggiornare la sezione in inglese (e far aggiornare le traduzioni in seguito).

Reply:
Capito! Grazie della spiegazione. Ho aggiornato l'inglese e creato una nuova PR (#450). Se va bene, posso rivedere questa traduzione!

Reply:
La PR contiene tutte le modifiche presenti in questa PR. Credo to abbia fatto `git checkout -b en-week06` da `it-translation-week06` e non da `master`. Aggiorniamo intanto questa in italiano. Per l'altra, vuoi fare un 
```
git checkout en-week06
git reset --hard master
git cherry-pick 88abf00
git push -f
```